dogstatsd/src/aggregator.rs

```diff
@@ -328,21 +318,29 @@ pub mod tests {
         metric_id: &str,
         value: f64,
         tags: &str,
+        timestamp: i64,
     ) {
         let aggregator = aggregator_mutex.lock().unwrap();
-        if let Some(e) =
-            aggregator.get_entry_by_id(metric_id.into(), &Some(SortedTags::parse(tags).unwrap()))
-        {
+        if let Some(e) = aggregator.get_entry_by_id(
+            metric_id.into(),
+            &Some(SortedTags::parse(tags).unwrap()),
+            timestamp,
+        ) {
```

Comment:
Is this a test or actual code? Lots of unwraps

Also this line is crazy

Reply:
lol it's a test method